crates/router/src/connector/adyen/transformers.rs

```diff
@@ -788,12 +788,13 @@ fn get_line_items(item: &types::PaymentsAuthorizeRouterData) -> Vec<LineItem> {
     let line_item = LineItem {
         amount_including_tax: Some(item.request.amount),
         amount_excluding_tax: Some(item.request.amount),
-        description: order_details.map(|details| details.product_name.clone()),
+        description: order_details
+            .map(|details| details.iter().map(|v| v.product_name.clone()).collect()),
```

Comment:
I guess this also should be sent as a `Vec<LineItem>`. Can @ArjunKarthik @jagan-jaya confirm?